Resources/Prototypes/Body/Organs/arachnid.yml

```diff
@@ -1,42 +1,17 @@
 - type: entity
   id: BaseArachnidOrgan
-  parent: BaseItem
   abstract: true
```

Comment:
I saw that you consistently paid attention to put parent before id, thank you. But all abstract definitions should also go before parent and id, the full sequence is `type > abstract > parent > id > categories > name > suffix > description > components.`